dvc/lock.py

```diff
@@ -26,10 +27,59 @@ class LockError(DvcException):
     """Thrown when unable to acquire the lock for dvc repo."""
 
 
+class Lock(object):
+    """Class for dvc repo lock.
+
+    Uses zc.lockfile as backend.
+    """
+
+    def __init__(self, lockfile, friendly=False, **kwargs):
+        self._friendly = friendly
+        self.lockfile = lockfile
+        self._lock = None
+
+    @property
+    def files(self):
+        return [self.lockfile]
+
+    def _do_lock(self):
+        try:
+            with Tqdm(
+                bar_format="{desc}",
+                disable=not self._friendly,
+                desc=(
+                    "If dvc froze, see `hardlink_lock` in "
```

Comment:
minor dvc -> DVC cc @jorgeorpinel

Reply:
Agree, or use back ticks if wanting to refer specifically to the `dvc` command.

Reply:
fixed